test/unit/InputObserver.spec.js

```diff
@@ -74,5 +75,27 @@ describe("InputObserver", function () {
       // Then
       expect(this.inst.get(inputType)).to.be.eql({"y": 200});
     });
+    it("should check delta that there is no bounce and the position is out", done => {
+      // Given
+      const inputType = {
+        axes: ["y"]
+      };
+      let y = 50;
+      this.axes.setTo({y: 50}, 0);
+      this.axes.on("change", ({pos, delta}) => {
+        // Then
+        // Find the value as approximated as possible due to floating decimal 11point problems.
+        expect(delta.y + y).to.be.closeTo(pos.y, 0.0000001);
```

Comment:
@younkue Is it related with https://github.com/naver/egjs-axes/issues/116#issuecomment-432494756?
But Does `closeTo` means  approximated value? So Is follow fine, Isn't it?
```js
 expect(delta.y + y).to.be.closeTo(pos.y, 0);
```

Reply:
`pos.y` is not a problem, but `delta.y + y` has  a floating point problem.

Reply:
@younkue I got it!